CHANGELOG.md

```diff
@@ -20,6 +20,29 @@ All notable changes to this project will be documented in this file, in reverse
 
 - Nothing.
 
+## 3.0.0 - 2016-12-??
+
+### Added
+
+- [#23](https://github.com/zendframework/zend-expressive-helpers/pull/23) adds support to UrlHelper for generating
+  default route w/o parameters
+- [#9](https://github.com/zendframework/zend-expressive-helpers/pull/9) UrlHelper passes `$routerOptions` to underlying
+  router, if provided.
+- [#27](https://github.com/zendframework/zend-expressive-helpers/pull/27) adds query params and fragment identifier
+   support to `UrlHelper`
```

Comment:
I'm thinking you should detail what's *breaking* about any of these changes — i.e., what changes are forcing the new major (vs minor) version. That way anybody who was _extending_ any classes in the library knows what they may need to change in order to adopt this version, and anybody who was _consuming_ understands what changes they may need to make when invoking functionality from them.